github/ci/prow/files/job-config.yaml

```diff
@@ -112,3 +148,268 @@ presubmits:
         resources:
           requests:
             memory: "500Mi"
+
+  kubevirt/kubevirtci:
+  - name: check-provision-os-3.11.0-multus
+    always_run: false
+    optional: true
+    decorate: true
+    max_concurrency: 1
+    spec:
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: In
+                values:
+                - bare-metal
+      containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - "cd os-3.11-multus && ./provision.sh"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "5Gi"
+
+  - name: check-provision-os-3.11.0
+    always_run: false
+    optional: true
+    decorate: true
+    max_concurrency: 1
+    spec:
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: In
+                values:
+                - bare-metal
+      containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - "cd os-3.11 && ./provision.sh"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "5Gi"
+
+  - name: check-provision-os-3.11.0-crio
+    always_run: false
+    optional: true
+    decorate: true
+    max_concurrency: 1
+    spec:
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: In
+                values:
+                - vm
+              - key: kubernetes.io/hostname
```

Comment:
Why is this selector is different from the others?

Reply:
Evgheni enable nested virt on that node for me, to test if the privisioning of the clusters, which does not involve starting nested vms, could be done in vms instead of bare-metal. For k8s clusters it looks like an option (have seen 30 minutes instead of 10 minutes), for openshift I think it is too slow (more than two hours compared to 50 minutes).